src/core/core.plugins.js

```diff
@@ -167,18 +169,23 @@ export default {
 		cache.descriptors = descriptors;
 		cache.id = this._cacheId;
 		return descriptors;
-	},
+	}
 
 	/**
 	 * Invalidates cache for the given chart: descriptors hold a reference on plugin option,
 	 * but in some cases, this reference can be changed by the user when updating options.
 	 * https://github.com/chartjs/Chart.js/issues/5111#issuecomment-355934167
 	 * @private
 	 */
-	_invalidate: function(chart) {
+	_invalidate(chart) {
 		delete chart.$plugins;
 	}
-};
+}
+
+// singleton instance
+const plugins = new PluginService();
```

Comment:
can you combine this line with the next one? Just `export default new PluginService();`

also, I was curious about it being instantiated here because we don't do that anywhere else. If you import this multiple places will you only get a single instance?

Reply:
I'm not sure combining the lines works the same way. 
I used this already in `Animator` to make it singleton (handling all the charts in a page)